app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    pass

```

Comment:
You should add a custom error message to your `OutdatedVaccineError` class. A descriptive error message can help other developers understand what went wrong when the exception is raised.